documentation/components/tutorial-flow-components-setup.asciidoc

```diff
@@ -53,57 +59,21 @@ Using the platform dependency guarantees that you'll get versions of Flow and th
     </dependency>
 </dependencies>
 ----
-[NOTE]
-`vaadin.platform` releases regularly and you can check the version number https://github.com/vaadin/platform/releases[here]
 
 [NOTE]
-The components are also provided as single dependencies per component. See https://github.com/vaadin/platform/blob/master/vaadin/pom.xml#L68[the platform pom.xml] for example.
-
-The platform also contains ready made _theming_ for the components that help you tweak and tune the look and feel of the components.
-See the <<../theme/using-component-themes#,Using Component Themes>> for more info.
-
-For more ready-made Java APIs for Web Components, you should check the https://vaadin.com/directory/search?framework=Vaadin%2010[Vaadin Directory.]
-
-= Flow Component Package Contents Explained
-
-A Flow component package consists of two main sections:
-
-* Java API classes
-* Web Components' files (html, js, css etc)
-
-All sections are used by Flow to display the components in web pages correctly.
-By default, both sections are provided in the same jar.
-
-Web components' files are provided as https://github.com/webjars/webjars/[webjars] –
-jar files that contain webcomponents' files,
-heavily influenced by current Polymer package manager: https://bower.io/[bower]
-(not developed or maintained by Vaadin)
-
-Webjars are designed to replace `bower` usage by using `Maven` for the same purposes.
+It is possible to include components individually by declaring a dependency for each component. See below for an example.
 
-`Flow` is able to resolve requests into webjars' contents (if intended by request),
-imitating the regular web component's files.
+The platform also includes theming for the components that you can edit to ensure the styling suits your UI. See <<../theme/using-component-themes#,Using Component Themes>> for more.
 
-Although currently enabled by default, webjars do not restrict users from using external
-web components' files:
+You can find additional prebuilt Web Components (with Java APIs) contributed by the Vaadin community in the https://vaadin.com/directory/search?framework=Vaadin%2010[Vaadin Directory.]
```

Comment:
@samie What should be the link here? Should we use Vaadin 10 or something else as a filter🤔

Reply:
Removed the search filter